### ISSUE TITLE
Split train and validation set by function

### ISSUE THREAD

**Issue by just-hjkwon** [just-hjkwon/pytorch_tools#1]

**Comment by just-hjkwon** [comment]
Merged.
